sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/command/CommandProcessor.java

```diff
@@ -74,6 +77,7 @@ public static void process(String command) {
             return;
         }
         String commandArgs = commandInfo.length > 1 ? commandInfo[1] : null;
+        DynamicAgentArgsManager.getAgentArgs().putAll(agentArgsMap);
```

Comment:
putall methon should be implemented within DynamicAgentArgsManager

Reply:
no need here. getAgentArgs is a public method for plugin